Core/Core/Models/GraphTraversal/GraphTraversal.cs

```diff
@@ -73,7 +74,10 @@ public IEnumerable<T> Traverse(Base root)
       Base current = head.current;
       var activeRule = GetActiveRuleOrDefault(current);
 
-      foreach (string childProp in activeRule.MembersToTraverse(current))
+      var toTraverse = activeRule.MembersToTraverse(current).ToList();
+
+      _ = 0;
```

Comment:
???

Reply:
is this JBDD's breakpoint holder pattern?

Reply:
It is, good catch, I'll revert this 😁